src/domain/cards/customerProfileCard/CustomerProfileCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import CardHeader from '../../../common/cardHeader/CardHeader';
+import CardBody from '../../../common/cardBody/CardBody';
+import PrivateCustomerDetails, {
+  PrivateCustomerDetailsProps,
+} from './privateCustomerDetails/PrivateCustomerDetails';
+import OrganizationCustomerDetails, {
+  OrganizationCustomerDetailsProps,
+} from './organizationCustomerDetails/OrganizationCustomerDetails';
+
+export type CustomerProfileCardProps = {
+  className?: string;
+} & (PrivateCustomerDetailsProps | OrganizationCustomerDetailsProps);
+
+const renderDetails = (props: CustomerProfileCardProps) => {
+  if ('organization' in props) {
+    return <OrganizationCustomerDetails {...props} />;
```

Comment:
It seems that this branch is not entered correctly? Tried selecting customer `Anneli Parviainen` who was listed under customer group "company" and the page only rendered personal details.

Reply:
Same here, wasn't able to test it.

Reply:
Now this works.